charts/fluid/fluid/templates/webhook/webhook.yaml

```diff
@@ -41,6 +41,10 @@ spec:
             - name: VALUE_SERVERLESS_PLATFORM
               value: {{ .Values.webhook.serverlessInject.platformValue | quote }}
             {{- end }}
+            {{- if .Values.webhook.serverlessInject }}
```

Comment:
why not  using {{- if .Values.webhook.serverlessInject.vfuseResourceName }} ?

Reply:
Done